### PR TITLE
Add optuna dashboard, json_export and simplify graph generation

### PR DESCRIPTION
**Pull request by RomanSraj**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -2,8 +2,8 @@
 # We could directly refer to a commit here, but we probably shouldn't merge unstable features into master, anyway.
 # If you are not logged into GitHub while running this command, you may need to install the package manually
 # Reference Link: https://docs.readthedocs.io/en/stable/guides/private-python-packages.html
--e git+https://github.com/heidmic/suprb@main#egg=suprb
-# If the following throws an error along the lines:
+-e git+https://github.com/heidmic/suprb@master#egg=suprb
+# If the following throws an error along the lines:q
```
Comment (heidmic) [inline requirements.txt:6]: why q?

File: runs/solution_composition/solution_composition_tuning_updated.py [hunk 2]
```diff
@@ -156,28 +164,30 @@ def suprb_space(trial: Trial, params: Bunch):
             if isinstance(params.solution_composition__init, suprb.solution.initialization.RandomInit):
                 params.solution_composition__init__p = trial.suggest_float('solution_composition__init__p', 0.3, 0.8)
 
-            params.solution_composition__init__fitness = trial.suggest_categorical('solution_composition__init__fitness', ['PseudoBIC', 'ComplexityEmary', 'ComplexityWu'])  # nopep8
```
Comment (heidmic) [inline runs/solution_composition/solution_composition_tuning_updated.py:159]: remove the commented lines? we could do it in a separate PR though

File: README.md [hunk 3]
```diff
@@ -72,8 +72,3 @@ data_directory --- ES ------ concrete_strength
 
 Attention: **latex_tabulars** needs to specify a **summary_csv_dir** at the top of its file. (Not necessary if all scripts are executed at once) 
 
-
-## Run logging_output_scripts
```
[the remark this answers is not in the document]
Comment (RomanSraj) [reply README.md:76]: Created issue: https://github.com/heidmic/suprb-experimentation/issues/38

File: experiments/parameter_search/optuna.py [hunk 4]
```diff
@@ -55,7 +58,11 @@ def objective(trial: optuna.Trial):
 
         sampler = self._get_optimizer(self.tuner)(seed=self.random_state)
 
-        study = optuna.create_study(sampler=sampler)
+        storage_name = f'sqlite:///suprb_optuna_{datetime.now().strftime("%Y-%m-%d")}.db'
```
[the remark this answers is not in the document]
Comment (RomanSraj) [reply experiments/parameter_search/optuna.py:61]: Created issue: https://github.com/heidmic/suprb-experimentation/issues/39

File: experiments/parameter_search/optuna.py [hunk 5]
```diff
@@ -55,7 +58,11 @@ def objective(trial: optuna.Trial):
...
+        study = optuna.create_study(sampler=sampler,
+                                    study_name=self.study_name,
+                                    # storage=storage_name,
```
[the remark this answers is not in the document]
Comment (RomanSraj) [reply experiments/parameter_search/optuna.py:64]: Created issue: https://github.com/heidmic/suprb-experimentation/issues/40

File: logging_output_scripts/latex_tabulars.py [hunk 6]
```diff
@@ -2,21 +2,21 @@
...
 This script uses the tabulate package (https://pypi.org/project/tabulate/)
 to create LaTex tables based on the values calculated in Summary_csv.py
 (Except for Genomes-Tables which use a Json)
 """
+with open('logging_output_scripts/config.json') as f:
+    config = json.load(f)
 
-summary_csv_dir = "logging_output_scripts/outputs/csv_summary"
 final_output_dir = f"{config['output_directory']}"
-check_and_create_dir(final_output_dir, 'latex_tabular')
```
[the remark this answers is not in the document]
Comment (RomanSraj) [reply logging_output_scripts/latex_tabulars.py:16]: It's present in line 235 in `if __name__ == '__main__':`

File: experiments/parameter_search/optuna.py [hunk 7]
```diff
@@ -35,6 +37,7 @@ def __init__(
         self.callback = callback
         self.tuner = tuner
         self.timeout = timeout
+        self.study_name = study_name
```
[the remark this answers is not in the document]
Comment (heidmic) [reply experiments/parameter_search/optuna.py:40]: new issue: https://github.com/heidmic/suprb-experimentation/issues/41
